adafruit_ble/services/apple.py

```diff
@@ -147,31 +162,7 @@ def update(self, event_flags, category_id, category_count):
     def __str__(self):
         # pylint: disable=too-many-branches
         flags = []
-        category = None
-        if self.category_id == 0:
-            category = "Other"
-        elif self.category_id == 1:
-            category = "IncomingCall"
-        elif self.category_id == 2:
-            category = "MissedCall"
-        elif self.category_id == 3:
-            category = "Voicemail"
-        elif self.category_id == 4:
-            category = "Social"
-        elif self.category_id == 5:
-            category = "Schedule"
-        elif self.category_id == 6:
-            category = "Email"
-        elif self.category_id == 7:
-            category = "News"
-        elif self.category_id == 8:
-            category = "HealthAndFitness"
-        elif self.category_id == 9:
-            category = "BusinessAndFinance"
-        elif self.category_id == 10:
-            category = "Location"
-        elif self.category_id == 11:
-            category = "Entertainment"
+        category = NOTIFICATION_CATEGORIES[self.category_id]
```

Comment:
If `self.category_id` is out of bounds, it will raise an error, so maybe just leave it as None, like the original code?
```python
        category = NOTIFICATION_CATEGORIES.get(self.category_id, None)
```

Reply:
Tuples and lists don't have get so I check the length instead. Thanks!